pkg/scheduler/preemption/preemption_test.go

```diff
@@ -754,6 +809,89 @@ func TestPreemption(t *testing.T) {
 			},
 			wantPreempted: sets.New("/wl2"),
 		},
+		"preempt while borrowing in cohort": {
+			admitted: []kueue.Workload{
```

Comment:
add another workload in standard_b under its nominal quota, to make sure it's not preempted, even if it has lower priority.

Reply:
I've added the workload under b_best_effort, in standard I put workloads of higher priority, so it could be inconclusive why it wasn't preempted.